scripts/build-dist

```diff
@@ -22,6 +22,9 @@ done
 for whl in /dd-trace-py/dist/*-linux_${ARCH}.whl;
 do
   auditwheel repair "${whl}" -w /dd-trace-py/dist
+
+  # Remove linux wheel since we only want the manylinux wheels
+  rm "${whl}"
```

Comment:
Why not remove them from being built? 

https://github.com/DataDog/dd-trace-py/blob/master/scripts/build-dist#L15-L19

Reply:
we need to build them first to then convert them into a manylinux wheel

Reply:
build linux wheel -> convert to manylinux wheel

Reply:
👍